app/assets/stylesheets/actiontext.css

```diff
@@ -29,3 +29,9 @@
   padding: 0 !important;
   max-width: 100% !important;
 }
+
+/* Hide attachment button in Trix toolbar */
+trix-toolbar .trix-button--icon-attach {
```

Comment:
Think we need a better comment - we're doing this for a reason, hopefully its clear that we're hiding the button ;-) I guess this is temporary as don't support uploads yet?